src/EconomicStrategy/EconomicStrategyHelpers.ts

```diff
@@ -1,13 +1,17 @@
 import { IEconomicStrategy } from './IEconomicStrategy';
 import Config from '../Config';
+import { BigNumber } from '../../node_modules/bignumber.js';
```

Comment:
Can we shorten the import to just 'bignumber.js'?

Reply:
Yeah, this was auto generated by VScode